.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://registry.npmjs.org/
```

Comment:
Unrelated. It enforces `npm` to use the public registry when working with this project.